src/Apps/Consign/Routes/SubmissionFlow/Utils/validation.ts

```diff
@@ -47,6 +43,10 @@ export const artworkDetailsValidationSchema = yup.object().shape({
       coordinates: yup.array(yup.number()),
     })
     .required(),
+  category: yup
+    .string()
+    .required("Please select a category")
```

Comment:
I would probably say "Please select a _medium_" to match the common name (as well as the form label) for this field.